content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md

```diff
@@ -15,53 +16,45 @@ code to the rescue! We can now provision an entire EKS cluster with a
 CLI single gesture, thanks to [the `@pulumi/eks`
 package](https://github.com/pulumi/pulumi-eks). Let's see how.
 
-![EasyEKS](https://blog.pulumi.com/hs-fs/hubfs/EasyEKS.png?width=1800&name=EasyEKS.png){width="1800"
-sizes="(max-width: 1800px) 100vw, 1800px"
-srcset="https://blog.pulumi.com/hs-fs/hubfs/EasyEKS.png?width=900&name=EasyEKS.png 900w, https://blog.pulumi.com/hs-fs/hubfs/EasyEKS.png?width=1800&name=EasyEKS.png 1800w, https://blog.pulumi.com/hs-fs/hubfs/EasyEKS.png?width=2700&name=EasyEKS.png 2700w, https://blog.pulumi.com/hs-fs/hubfs/EasyEKS.png?width=3600&name=EasyEKS.png 3600w, https://blog.pulumi.com/hs-fs/hubfs/EasyEKS.png?width=4500&name=EasyEKS.png 4500w, https://blog.pulumi.com/hs-fs/hubfs/EasyEKS.png?width=5400&name=EasyEKS.png 5400w"}
+![Pulumi making EKS Easy](./easy-eks.png)
 
-To get started, download the free and open source [Pulumi
-SDK](https://pulumi.io/quickstart/install.html). The SDK includes
+To get started, download the free and open source
+[Pulumi SDK]({{< ref "/docs/quickstart" >}}). The SDK includes
```

Comment:
The actual CLI install page is now: `/docs/reference/install.md`.

I'll add a task for myself to go through all the posts and make sure our "get started" instructions all make sense and are relatively consistent (as I already need to do that elsewhere in the docs). The nice thing about having all the posts in Hugo is that this should be much easier, rather than manually clicking around in HubSpot to make edits.

Reply:
👍